### DISCUSSION TITLE
Error 429

### DISCUSSION THREAD

**Discussion by andreas-p** [bimmerconnected/bimmer_connected#456] [Q&A]
I'm polling the API for status changes. After a while, I encounter `Client error '429 Too Many Requests' for url 'https://cocoapi.bmwgroup.com/eadrax-vcs/v1/vehicles'`
I increased the poll period from 60 to 90 seconds, but after a while the problem will reappear.

Is there any way to avoid this?

**Comment by rikroe** [comment]
Setting it to at least 5 minutes for now, see https://github.com/home-assistant/core/issues/73370#issuecomment-1159060882. You could also keep one minute but on `httpx.HTTPStatusError` with status code 429 wait 5 minutes.
